bubblewrap.c

```diff
@@ -2216,6 +2237,13 @@ main (int    argc,
       prctl (PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &seccomp_prog) != 0)
     die_with_error ("prctl(PR_SET_SECCOMP)");
 
+  /* Optionally limit memory. This must occur after the fork and
+   * before the exec* call. */
+  if (opt_limit_mem) {
+    limit_mem.rlim_cur = limit_mem.rlim_max = opt_limit_mem * 1048576;
+    setrlimit(RLIMIT_DATA, &limit_mem);
```

Comment:
And yeah, we should probably just disable this in the setuid case...unless it seems easy to get the current limit and check it ourself.  (Alternatively, maybe we could temporarily drop `CAP_SYS_RESOURCE` before doing this call?  Actually, maybe we're dropping it already?)